packages/react/.storybook/webpack.config.js

```diff
@@ -48,13 +52,40 @@ const styleLoaders = [
   {
     loader: 'sass-loader',
     options: {
-      includePaths: [path.resolve(__dirname, '..', 'node_modules')],
-      data: `
-        $feature-flags: (
-          ui-shell: true,
-          enable-css-custom-properties: true,
-        );
-      `,
+      prependData(loaderContext) {
+        const entrypoint = path.resolve(__dirname, './_container.scss');
+        const flags = `
+          $feature-flags: (
+            ui-shell: true,
+            enable-css-custom-properties: ${CARBON_REACT_STORYBOOK_USE_CUSTOM_PROPERTIES},
+          );
+        `;
+
+        // If we're attempting to process our entrypoint, we'll just pass in the
```

Comment:
I’d like to reiterate on the record that this is a signal of an underlying problem with our reset and style loading. Is this something on the v11 wish list?

Reply:
@vpicone yeah, I think we'll have a defined load order for styles that should resolve this. This would mean that we specify here is the import for fonts, here is the import for the reset, etc. versus each component specifying that it needs the reset

Reply:
I don’t have the box note handy, something around the reset was in there right?

Reply:
@vpicone went ahead and dropped it over in: https://github.com/carbon-design-system/carbon/issues/4358#issuecomment-546504408